torch/distributed/device_mesh.py

```diff
@@ -135,6 +139,15 @@ def get_mesh_dim_by_name(
                 )
             return not_none(device_mesh.mesh_dim_names.index(mesh_dim_name))
 
+        def get_child_mesh(
```

Comment:
instead of creating a separate method, It think we should just put this logic into `create_child_mesh` directly